public/app/percona/shared/components/Form/FieldAdapters/Checkbox.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { HTMLProps, useCallback } from 'react';
```

Comment:
There's already a Checkbox component in platform-core, please use that one and remove this file. I did the same for the Table component in this shared folder if you want to check.

Reply:
Done, thanks for noticing it,  wasn't used anywhere, just missed this file deletion